tests/garage/torch/policies/test_context_conditioned_policy.py

```diff
@@ -0,0 +1,115 @@
+"""This is a script to test the ContextConditionedPolicy module."""
+
+import akro
+import numpy as np
+import pytest
+import torch
+import torch.nn as nn
+from torch.nn import functional as F  # NOQA
+
+from garage.envs.env_spec import EnvSpec
+from garage.tf.envs import TfEnv
+from garage.torch.embeddings import MLPEncoder
+from garage.torch.policies import ContextConditionedPolicy
+from garage.torch.policies import TanhGaussianMLPPolicy
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class TestContextConditionedPolicy:
+    """Test for ContextConditionedPolicy."""
+    # yapf: disable
+    @pytest.mark.parametrize('latent_dim, hidden_sizes, updates', [
+        (5, (3, 3, 3), 5),
+        (7, (5, 7, 5), 7),
+        (10, (9, 4, 2), 10),
+    ])
+    # yapf: enable
+    def test_module(self, latent_dim, hidden_sizes, updates):
```

Comment:
can you split this test into a setup phase followed by cases for each of your major assertions? that will make it easier to understand and maintain.